gradle.properties

```diff
@@ -42,6 +42,6 @@ systemProp.nessie.integrationsTesting.enable=true
 systemProp.sparkVersions=3.1,3.2,3.3
 systemProp.scalaVersion=2.12
 
-# System property to tell the included Iceberg build to build Flink 1.14 AND 1.15.
+# System property to tell the included Iceberg build to build Flink 1.14, 1.15 and 1.16.
```

Comment:
nit: probably not worth repeating in the comment here as well (since similarly the one for spark above is already outdated)